turbinia/api/client/turbinia_api_lib/rest.py

```diff
@@ -7,7 +7,6 @@
     Generated by: https://openapi-generator.tech
 """
 
-
```

Comment:
Why do we need to modify this file in this pull request? Seems unrelated to the Pydantic method changes needed.

Reply:
Reverted the changes to that file.